roles/openshift_logging/tasks/install_logging.yaml

```diff
@@ -66,7 +69,7 @@
     openshift_logging_elasticsearch_pvc_name: "{{ openshift_logging_es_pvc_prefix ~ '-' ~ item.2 if item.1 is none else item.1 }}"
     openshift_logging_elasticsearch_replica_count: "{{ openshift_logging_es_cluster_size | int }}"
 
-    openshift_logging_elasticsearch_storage_type: "{{ 'pvc' if openshift_logging_es_pvc_dynamic | bool else 'emptydir' }}"
+    openshift_logging_elasticsearch_storage_type: "{{ 'pvc' if openshift_logging_es_pvc_dynamic | bool else 'pvc' if openshift_hosted_logging_storage_kind == 'nfs' else 'emptydir' }}"
```

Comment:
would this achieve the same?
`"{{ 'pvc' if (openshift_logging_es_pvc_dynamic or openshift_hosted_logging_storage_kind == 'nfs') else 'emptydir' }}"`